src/client/components/componentWithSliders/allSliders/allBlocks.js

```diff
@@ -1,24 +1,23 @@
 import React, { Component } from 'react';
-import './alllBlocks.css';
-
 import Pizzicato from 'pizzicato';
+
 import BlockOfSliders from './blockOfSliders';
+
+
 import Button from './button';
 import { pauseIcon, playIcon, stopIcon } from '../../../assets/icons/icons';
-
 import { BLOCKS } from '../../../helpers/constants';
 import Sound from '../../../sounds/sound.mp3';
-
+import './alllBlocks.css';
 
 class AllBlocks extends Component {
   constructor(props, context) {
     super(props, context);
     this.state = {
-      sound: new Pizzicato.Sound(Sound,
-        () => BLOCKS.forEach(({ createEffect }) => {
-          const { sound } = this.state;
-          sound.addEffect(createEffect);
-        })),
```

Comment:
I would suggest to move callback function to a separate function, see 17 line

Reply:
done)